### DISCUSSION TITLE
Is there any superset maximum specification for each object?

### DISCUSSION THREAD

**Discussion by victarary** [apache/superset#15704] [Q&A / Help] [answered]
Hi Buddies,
I'm currently investigating on some maximum specifications in superset, is there any specification in superset to describe following limitations?

1.How many roles an user can be maximumly assigned?
2.How many permissions a role can be maximumly assigned?

**Comment by nytai** [comment, marked as answer]
i don't think there's a hard limit, but as these are used for permissions checks in almost every request they do impact performance linearly.
